src/wm/background.vala

```diff
@@ -140,6 +140,71 @@ public class BudgieBackground : Meta.BackgroundGroup
         }
     }
 
+    /**
+     * call accountsservice dbus with the background file name
+     * to update the greeter background if the display
+     * manager supports the dbus call.
+     */
+    void set_accountsservice_user_bg(string background)
+    {
+        DBusConnection bus;
+        Variant variant;
+
+        try
+        {
+            bus = Bus.get_sync(BusType.SYSTEM);
+        }
+        catch (IOError e)
+        {
+            warning("Failed to get system bus: %s", e.message);
+            return;
+        }
+
+        try
+        {
+            variant = bus.call_sync(
+                "org.freedesktop.Accounts",
```

Comment:
Can just create a schema const to reference like `ACCOUNTS_SCHEMA` and save the multiple identical strings.